sdk/cosmosdb/cosmos/src/utils/batch.ts

```diff
@@ -186,3 +187,13 @@ export function decorateOperation(
   }
   return operation as Operation;
 }
+
+function deepFind<T, P extends string>(document: T, path: P): JSONObject {
```

Comment:
Might be worth adding some unit tests for this beyond the integration test

Reply:
added